scripts/winesapos-setup.sh

```diff
@@ -67,9 +67,28 @@ else
 fi
 echo "Turning on the Mac fan service if the hardware is Apple complete."
 
+# Dialog to ask the user what mirror region they want to use
+# Fetch the list of regions from the Arch Linux mirror status JSON API
+arch_mirror_regions=$(curl -s https://archlinux.org/mirrors/status/json/ | \
+                        grep -oP '(?<="country_code": ")[^"]*' | sort | uniq) # grep maybe fragile, woule be better if we can use jq
```

Comment:
Good idea on switching to `jq`! I tested this out and it works at a drop-in replacement:

```
curl -s https://archlinux.org/mirrors/status/json/ | jq -r .urls[].country_code | sort | uniq
```

However, the format of both the old and new commands are off. `kdialog` is not listing each individual country code but actually displaying all of them as the only option. Maybe we need to inject newline characters? I'm not sure. We'll need to test with this some more.

For comparison, the output of the listed time zones here works: https://github.com/LukeShortCloud/winesapOS/blob/3.3.0/scripts/winesapos-setup.sh#L304

Reply:
`jq` is only installed on the performance and secure images (not the minimal image). It is a dependency of EmuDeck. I'll make the necessary changes to add that into the minimal image after we get this merged in.

Reply:
Alright, I switched to `jq` for parsing